tests/integration/ambient/baseline_test.go

```diff
@@ -1258,6 +1258,77 @@ spec:
 	})
 }
 
+// Relies on the suite running in a cluster with a CNI which enforces K8s netpol but presently has no check
+func TestK8sNetPol(t *testing.T) {
+	framework.NewTest(t).
+		Features("security.reachability").
+		Run(func(t framework.TestContext) {
+			t.Skip("This did help find a bug so probably has value but skipping for now")
```

Comment:
And if skipping, we should justify why we're skipping.. ideally with a link to an issue.

Reply:
It'll require a bit more work if we want to enable it now because the test can't run in the regular ambient integ suite anymore since it is back to using KinD's default CNI. We'd need to gate the test on having another CNI installed.

Reply:
Okay then I'm fine opening an issue to do that next and skipping with that issue reference.